### ISSUE TITLE
Update demo gif in `README.md`

### ISSUE THREAD

**Issue by kyu08** [dlvhdr/gh-dash#415]
The new UI released on [v4.5.0](https://github.com/dlvhdr/gh-dash/releases/tag/v4.5.0) is awesome! I love it!
But maybe only small number of people recognize the awesome update.

So please update demo gif in `README.md` when you have time.

I tried to fix by myself, but the config file used for demo [seems not public.](https://github.com/dlvhdr/gh-dash/blob/c524f60c308dd13ce0c1f9feee5cb6aa79862951/demo.tape#L11)

**Comment by dlvhdr** [comment]
Yeah I would like that but currently it's a bit cumbersome, because the app does network requests and that's hard to record.
I would like to have some kind of mocking in place to make the process easier.
If you would like to update it yourself, feel free to create a demo config you think is reasonable and record the same actions I'm doing.

**Comment by kyu08** [comment]
> Yeah I would like that but currently it's a bit cumbersome, because the app does network requests and that's hard to record.

I just want to make sure I understand you correctly, you mean a demo with network request using vhs is hard for some reason?

**Comment by dlvhdr** [comment]
It's just something I want to automate and not rely on the data from the network. I think the demo  gif showed my PRs which could be empty/have short titles/not something I can share as they're work related or can't comment on.
